src/raftstore/store/store.rs

```diff
@@ -743,6 +743,9 @@ impl<T: Transport, C: PdClient> Store<T, C> {
                 // Maybe split, but not registered yet.
                 if util::is_first_vote_msg(msg) {
                     self.pending_votes.push(msg.to_owned());
+                    info!("[region {}] region not exist but receive first vote msg",
```

Comment:
`[region {}] doesn't exist yet, wait for it to be split." seems more expressive.